docs/StorageClass.md

```diff
@@ -65,84 +58,47 @@ spec:
 
-Please see the Kubernetes
-[GKE PD documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd)
-for additional Storage Class configuration options that are available.
+For additional Storage Class configuration options, see the Kubernetes [GKE PD documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#gce-pd).
 
 
 ## Local Persistent Volumes
 
-Users of Kubernetes 1.14 or later may use
-[Local Persistent Volumes](https://kubernetes.io/blog/2019/04/04/kubernetes-1.14-local-persistent-volumes-ga/)
-with their Splunk cluster. The use of local, direct-attached storage can offer
-performance on par with running Splunk on bare metal, at the cost of
-sacrificing high availability provided by other storage options. Splunk’s
-built-in clustering technologies mitigate this cost by replicating your
-data across multiple instances.
+Users of Kubernetes 1.14 or later can also use Kubernetes [Local Persistent Volumes](https://kubernetes.io/blog/2019/04/04/kubernetes-1.14-local-persistent-volumes-ga/). The use of local, direct attached storage can offer performance similar to running Splunk Enterprise on bare metal, at the cost of sacrificing the high availability provided by other storage options. The built-in clustering technologies provided by Splunk Enterprise can mitigate the cost by replicating your data across multiple instances.
 
-If you are interested in using Local Persistent Volumes, we recommend
-considering these open source projects:
+If you are interested in using Local Persistent Volumes, consider one of these open source projects:
 
 * [TopoLVM](https://blog.kintone.io/entry/topolvm) (uses LVM to dynamically provision and manage volumes)
 * [Local Path Provisioner](https://github.com/rancher/local-path-provisioner) (shares a single, mounted volume on each node)
 
 
-## Additional Providers
+## Additional Storage Providers
 
-The [introduction](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/)
-of Kubernetes’s [Container Storage Interface](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) (CSI)
-has made it easy for new vendors to offer innovative solutions for managing
-the persistence of containerized applications. Users of Kubernetes 1.13 or
-later are encouraged to take a look at
-[this list of CSI Drivers](https://kubernetes-csi.github.io/docs/drivers.html).
+The [introduction](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/) of Kubernetes’s [Container Storage Interface](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) (CSI) has made it easy for new vendors to offer innovative solutions for managing the persistence of containerized applications. Users of Kubernetes 1.13 or later are encouraged to take a review the Kubernetes [list of CSI Drivers](https://kubernetes-csi.github.io/docs/drivers.html).
```

Comment:
nit: "encouraged to take a review the Kubernetes" -> "encouraged to review the Kubernetes"